bndtools.core.test/src/bndtools/core/test/launch/NullContextPresentationEngine.java

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2015 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
```

Comment:
This whole repo is dual licensed under AL2 OR EPL2. Not sure why you have this license header here.

Reply:
This is a cut-and-paste error. NullContextPresentationEngine started off life as a clone of https://github.com/eclipse/eclipse.platform.ui/blob/master/tests/org.eclipse.e4.ui.tests/src/org/eclipse/e4/ui/tests/application/HeadlessContextPresentationEngine.java